source/locale/pt_BR/LC_MESSAGES/internal-communication.po

```diff
@@ -32,8 +31,13 @@ msgid "Components"
 msgstr "Componentes"
 
 #: ../../source/internal-communication.rst:10
-msgid "The main components that are currently in `Dojot organization`_ are:numref:`dojot_components`."
-msgstr "Os principais componentes que estão atualmente na 'organização Dojot'_ são:numref:'dojot_components'."
+#, fuzzy
+msgid ""
+"The main components that are currently in `dojot platform`_ are "
```

Comment:
`dojot platform`_ link goes to where? 
Change to in  dojot platform or  just dojot

Reply:
I prefer only dojot